lib/xlsx/xform/book/workbook-xform.js

```diff
@@ -127,7 +127,7 @@ utils.inherits(WorkbookXform, BaseXform, {
       case 'workbook':
         this.model = {
           sheets: this.map.sheets.model,
-          properties: this.map.workbookPr.model,
+          properties: this.map.workbookPr.model || {},
```

Comment:
Looks good - could you also add a unit test that catches this issue?
in: spec/unit/xlsx/xform/book/workbook-xform.spec.js
best approach woud be making copies of the spec/unit/xlsx/xform/book/data/book.1.* files and adding a new expectation